numpy/_core/fromnumeric.py

```diff
@@ -2643,6 +2665,202 @@ def all(a, axis=None, out=None, keepdims=np._NoValue, *, where=np._NoValue):
                                   keepdims=keepdims, where=where)
 
 
+def _cumulative_func(x, func, axis, dtype, out, include_initial):
+    x = np.atleast_1d(x)
+    x_ndim = ndim(x)
```

Comment:
Good catch on `atleast_1d`. But now using `ndim` is no longer required, so I would just replace with `x.ndim`

Reply:
Done!